src/main.cpp

```diff
@@ -49,9 +49,15 @@ int main( int argc, char *argv[]) {
 
     if(!mq.is_only_instance()) {
         std::string msg;
+        std::string cmd;
+
         if(argc == 1) {
-            mq.SendMessage("kill");
-        } else if(static_cast<std::string>(argv[1]) == "help") {
+            cmd = "kill";
+        } else {
+            cmd = static_cast<std::string>(argv[1]);
```

Comment:
Better yet `cmd = argv[1]` in this case since the assignment operator for string can take a const char*